.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Tensor Comprehensions Github Issues Guidelines
+----------------------------------------------
+
+If you have a feature request or a bug report (build issue), please open an issue on Github and fill the template below so we can help you better and faster. If you have some general
+questions about Tensor Comprehensions, please visit our [slack channel](https://tensorcomprehensions.herokuapp.com/) or email us at tensorcomp@fb.com
+
+For build issues, please add `[Build]` at the beginning of issue title.
+
+When submitting a bug report, please include the following information (where relevant):
+- OS:
+- How you installed TC (docker, conda, source):
+- Python version:
+- CUDA/cuDNN version:
+- Conda version (is using conda):
+- Docker image (if using docker):
+- GCC/GXX version (if compiling from source):
+- Add LLVM/Tapir version:
```

Comment:
you literally copied "_Add_ LLVM/Tapir version" :)

Reply:
hahahaha